.github/styles/Google/Headings.yml

```diff
@@ -0,0 +1,61 @@
+extends: capitalization
+message: "'%s' should use sentence-style capitalization."
```

Comment:
Do we have agreement that this is the style we want?  I would vote yes, actually, but we don't have a style guide.  Perhaps you are adopting an external style guide (like Google's?). That is fine, especially assuming that we can modify this later if people want a different style, but it needs to be recorded someplace.

Reply:
Should this list of exceptions include some Keptn terms, like the KEPTN string itself?  What about Control Plane and Execution Plane, et cetera?  Again, I don't know that we have community consensus about capitalization and some terms -- the current docs are inconsistent.  I don't think you should get bogged down figuring out capitalization for Keptn terms.  Perhaps file an issue to determine which Keptn terms require capitalization and then augment this list?  That keeps the issue alive but doesn't slow you down.

Reply:
Yeah, @StackScribe for the time being I have adopted the Google style guide because my main motivation is to integrate the Vale GitHub Action into the docs engine. Yeah, I think we can have a separate discussion about which style guide to follow or we can have our custom style guide.

Yeah, I have added a few words to the exceptions list as of now. I did not have much idea about the whole content of current keptn docs so not sure what words should I add but yeah this work will go on forever.